templates/veoibd.yaml

```diff
@@ -27,6 +31,29 @@ Parameters:
     ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
 
 Resources:
+  MyCloudWatchLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub '${AWS::Region}-${AWS::StackName}-CloudWatchLogGroup'
```

Comment:
do not set name if not required.  let cloudformation manage names.

Reply:
The BatchJobQueue name is not required either. Should I remove that?